docs/administrator/karmadactl/register.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Karmadactl Register
+---
+
+`karmadactl register` is used to regiter member clusters to Karmada control plane with PULL mode. It is like `kubeadm join` that joins nodes to Kubernetes control plane.
+
+> Note: currently it only support the Karmada control plane that installed by `karmadactl init`.
+
+## Create bootstrap token in Karmada control plane
+
+In Karmada control plane, we can use `karmadactl token create` command to create bootstrap tokens whose default ttl is 24h.
+
+```
+$ karmadactl token create --print-register-command --kubeconfig /etc/karmada/karmada-apiserver.config
+```
+
+```
+# The example output is shown below
+karmadactl register 10.10.103.67:32443 --token t2jgtm.9nybj0526mjw1jbf --discovery-token-ca-cert-hash sha256:f5a5a43869bb44577dba582e794c3e3750f2050d62f1b1dc80fd3d6a371b6ed4
+```
+
+More details about `bootstrap token` please refer to:
+- [authenticating with bootstrap tokens](https://kubernetes.io/docs/reference/access-authn-authz/bootstrap-tokens/)
+
+## Execute `karmadactl register` in the member clusters
+
+In the member clusters, we can directly execute the above output `karmadactl register` command.
```

Comment:
Does `In the member cluster` mean on any node of a member cluster?

At the end of this doc, we mention the `current-context` thing, i guess It would be better to tell people we need a kubeconfig file.

Reply:
It is better to execute the command in the Kubernetes control plane of member clusters.